app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -140,6 +140,20 @@ def user_provided_data_map
           }
         end
 
+        def state_id_number
+          south_carolina_zero_padding
+
+          applicant.state_id_data.state_id_number
+        end
```

Comment:
another idea:
```suggestion
        def state_id_number
          case applicant.state_id_data.state_id_jurisdiction
          when 'SC'
            applicant.state_id_data.state_id_number.rjust(8, '0')
          else
            applicant.state_id_data.state_id_number
          end
        end
```

Reply:
I think I prefer this approach since it avoids mutating the PII

Reply:
Thanks, resolved in 0acd6849b